### PR TITLE
Add errorboundary aa1 to implement 1

### LINKED ISSUE #67: There should be a error boundary
**Describe the feature or change you'd like**
Currently when a component produces a error everything crashes. We should implement some kind of boundary which catches all errors of included components.

Also when a error occurs there should be a fancy error component which shows:
- 696 - Oops.. something went wrong
- The error
- The error stack

### LINKED ISSUE #68: The overview tab should use the error boundary
**Describe the feature or change you'd like**
The overview tab is the place where most errors occurs. Therefore we should wrap it with the error boundary.

**Additional context**
Ref: Error Boundary #67

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no error boundary component. #67 
- Currently the overview tab error are not handled and therefore it is very unstable. #68

**What is the new behavior (if this is a feature change)?**
- Now there is a error boundary component with a fancy error styling. #67 
- Now the overview tab uses the error boundary to handle errors. #68

**Other information**:
- 🐍

### CODE REVIEW

File: src/components/atoms/ErrorBoundary/index.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    if (this.state.hasError) {
+      if (!process.env.NODE_ENV || process.env.NODE_ENV === "development") {
+        //> Development Code
+        return (
+          <MDBAlert color="danger">
+            <code>{"696 - Oops, something went wrong :("}</code>
+            <div class="pl-2">
+              <p class="pt-3">{this.state.error.toString()}</p>
+
```
Comment (pinterid) [inline src/components/atoms/ErrorBoundary/index.jsx]: Remove newline.
```suggestion
```

File: src/components/atoms/ErrorBoundary/index.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    return this.props.children;
+  }
+}
+//#endregion
+
+//#region > PropTypes
+ErrorBoundary.propTypes = {
+  hidden: PropTypes.bool,
+};
+//#endregi
```
Comment (pinterid) [inline src/components/atoms/ErrorBoundary/index.jsx]:
```suggestion
//#endregion
```

File: src/components/atoms/ErrorBoundary/index.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        );
+      } else {
+        //> Production Code
+        return (
+          <MDBAlert color="danger">
+            <code>{"696 - Oops, something went wrong :("}</code>
+          </MDBAlert>
+        );
+      }
+    }
```
Comment (pinterid) [inline src/components/atoms/ErrorBoundary/index.jsx:76]:
```suggestion
    }
    
```

File: src/components/organisms/tabs/OverviewTab/index.jsx [hunk 4]
```diff
@@ -70,118 +71,123 @@ class OverviewTab extends React.Component {
...
-                <p className="lead">Pinned</p>
-              </MDBCol>
-              <MDBCol md="6 text-right">
-                <span className="clickable text-muted">Customize</span>
-              </MDBCol>
-            </MDBRow>
-            <MDBRow className="pinned">
-              {pinned.map((item, i) => {
-                return <Pinned {...item} key={i} />;
+        {/* <ErrorBoundary hidden="true"> */}
```
Comment (pinterid) [inline src/components/organisms/tabs/OverviewTab/index.jsx:74]: Can this be removed?
```suggestion
```

**Review by pinterid — CHANGES_REQUESTED** [review]
Some changes have to be done.

File: src/components/organisms/tabs/OverviewTab/index.jsx (hunk 4 above)
Comment (schettn) [reply src/components/organisms/tabs/OverviewTab/index.jsx:74]: No. I left this there to show the hidden flag.

**Review by pinterid — APPROVED** [review]
Looks good.

**Review by Aichnerc — APPROVED** [review]
LGTM... hue hue 69 hue hue

**Pull request merged**